src/Snowflake/Command.php

```diff
@@ -140,6 +195,28 @@ public static function cloneDatabaseFromShared(Connection $connection, array $da
                 foreach ($tables as $table) {
                     $tableName = $table['name'];
 
+                    $tableGrants = array_filter(
+                        $tablesGrants,
+                        function (array $v) use ($database, $schemaName, $tableName) {
+                            $validSchema = [
+                                sprintf('%s.%s.%s', $database, $schemaName, $tableName),
+                                sprintf('%s.%s.%s', $database, $schemaName, QueryBuilder::quoteIdentifier($tableName)),
+                                sprintf('%s.%s.%s', $database, QueryBuilder::quoteIdentifier($schemaName), $tableName),
+                                sprintf(
+                                    '%s.%s.%s',
+                                    $database,
+                                    QueryBuilder::quoteIdentifier($schemaName),
+                                    QueryBuilder::quoteIdentifier($tableName)
+                                ),
+                            ];
+                            return in_array($v['name'], $validSchema);
+                        }
+                    );
+                    $ownershipOnTable = current(array_filter($tableGrants, fn($v) => $v['privilege'] === 'OWNERSHIP'));
```

Comment:
to stejné, měla by být kontrola, že zbyde právě jeden řádek po filtru

Reply:
https://github.com/keboola/project-migration-tool/pull/1/commits/e3ef4bd0b54b6a1d6e78be2fa2c383df436f6657